src/Components/ImageCapture.js

```diff
@@ -0,0 +1,7 @@
+document
```

Comment:
lets change this to a clickable callable function maybe. This is pretty old style Javascript, it "works" but we can be cleaner :)

Reply:
Ok, I will change and re-submit

Reply:
For now yes, we can find the better way to do this when needed. Pretty sure the correct way will be a useEffect function

Reply:
yes, this should be a useEffect function within the camera capture component

Reply:
we can go over the changes in another pr as well for optimizations